doc/content/devices/models/quectel-kg200z/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Quectel KG200ZABTB"
+description: ""
+weight: 
+---
+
+{{< figure src="kg200z.png" alt="Quectel KG200ZABTB" class="float plain" width="60%">}}
+
+The [Quectel KG200ZABTB](https://www.quectel.com/product/lora-kg200z) is a high-performance LoRa module launched by Quectel, which supports ultra-low power consumption and long-range wireless transmission applications. It integrates an ARM Cortex-M4 core with the modulations of LoRa,(G)FSK, (G)MSK and BPSK, supporting LoRaWAN standard protocol and 470–510MHz, 862–928MHz LoRa frequency bands. Additionally, it incorporates AES hardware encryption for enhanced security. KG200Z features a compact form factor of 12.0mm × 12.0mm × 1.8mm and an LGA package to ensure seamless embedding of the module into size-constrained applications and reliable connectivity for these applications.
+
+This evaluation board (EVK) allows for programming of the module itself. For this guide though, we are just focussing on connecting it to {{% tts %}}
```

Comment:
What do you mean by "this evaluation board"? Which board? I only see described KG200Z module. Is there a link to the product? It would be also good to rephrase this a bit

Reply:
There is a Mouser page for it here: https://mouser.com/ProductDetail/Quectel/KG200ZABTB?qs=dbcCsuKDzFU6qVuvnq%252Bzmg%3D%3D This is the EVK